beacon-chain/db/kv/lightclient_test.go

```diff
@@ -2,221 +2,182 @@ package kv
-			Header: &ethpbv2.LightClientHeaderContainer_HeaderDeneb{
-				HeaderDeneb: &ethpbv2.LightClientHeaderDeneb{
-					Beacon: &ethpbv1.BeaconBlockHeader{
-						Slot:          1,
-						ProposerIndex: 1,
-						ParentRoot:    []byte{1, 1, 1},
-						StateRoot:     []byte{1, 1, 1},
-						BodyRoot:      []byte{1, 1, 1},
-					},
-					Execution: &enginev1.ExecutionPayloadHeaderDeneb{
-						FeeRecipient: []byte{1, 2, 3},
-					},
-					ExecutionBranch: [][]byte{{1, 2, 3}, {4, 5, 6}},
-				},
-			},
-		},
-		NextSyncCommittee: &ethpbv2.SyncCommittee{
-			Pubkeys:         nil,
-			AggregatePubkey: nil,
-		},
-		NextSyncCommitteeBranch: nil,
-		FinalizedHeader: &ethpbv2.LightClientHeaderContainer{
-			Header: &ethpbv2.LightClientHeaderContainer_HeaderDeneb{
-				HeaderDeneb: &ethpbv2.LightClientHeaderDeneb{
-					Beacon: &ethpbv1.BeaconBlockHeader{
-						Slot:          1,
-						ProposerIndex: 1,
-						ParentRoot:    []byte{1, 1, 1},
-						StateRoot:     []byte{1, 1, 1},
-						BodyRoot:      []byte{1, 1, 1},
-					},
-					Execution:       nil,
-					ExecutionBranch: nil,
-				},
-			},
-		},
-		FinalityBranch: nil,
-		SyncAggregate:  nil,
-		SignatureSlot:  7,
-	}
+	update, err := createUpdate(t, version.Deneb)
+	require.NoError(t, err)
 	period := uint64(1)
-	err := db.SaveLightClientUpdate(ctx, period, &ethpbv2.LightClientUpdateWithVersion{
-		Version: version.Deneb,
-		Data:    update,
-	})
+	err = db.SaveLightClientUpdate(ctx, period, update)
 	require.NoError(t, err)
 
 	retrievedUpdate, err := db.LightClientUpdate(ctx, period)
 	require.NoError(t, err)
-	require.DeepEqual(t, update, retrievedUpdate.Data, "retrieved update does not match saved update")
+	require.DeepEqual(t, update, retrievedUpdate, "retrieved update does not match saved update")
 }
 
 func TestStore_LightClientUpdates_canRetrieveRange(t *testing.T) {
 	db := setupDB(t)
 	ctx := context.Background()
-	updates := []*ethpbv2.LightClientUpdateWithVersion{
-		{
-			Version: 1,
-			Data: &ethpbv2.LightClientUpdate{
-				AttestedHeader:          nil,
-				NextSyncCommittee:       nil,
-				NextSyncCommitteeBranch: nil,
-				FinalizedHeader:         nil,
-				FinalityBranch:          nil,
-				SyncAggregate:           nil,
-				SignatureSlot:           7,
-			},
-		},
-		{
-			Version: 1,
-			Data: &ethpbv2.LightClientUpdate{
-				AttestedHeader:          nil,
-				NextSyncCommittee:       nil,
-				NextSyncCommitteeBranch: nil,
-				FinalizedHeader:         nil,
-				FinalityBranch:          nil,
-				SyncAggregate:           nil,
-				SignatureSlot:           8,
-			},
-		},
-		{
-			Version: 1,
-			Data: &ethpbv2.LightClientUpdate{
-				AttestedHeader:          nil,
-				NextSyncCommittee:       nil,
-				NextSyncCommitteeBranch: nil,
-				FinalizedHeader:         nil,
-				FinalityBranch:          nil,
-				SyncAggregate:           nil,
-				SignatureSlot:           9,
-			},
-		},
+	updates := make([]interfaces.LightClientUpdate, 0)
```

Comment:
```suggestion
	var updates []interfaces.LightClientUpdate
```
Plain variable declararation should be better than using `make()` ig

Reply:
fixed in https://github.com/prysmaticlabs/prysm/pull/14531/commits/03fd48cc4aa7f85fe96fe92624aff15ae09b71b2